src/config.ts

```diff
@@ -144,25 +172,31 @@ export interface Config {
   samplingRate?: number;
 
   /**
-   * Specifies how to use incoming trace context headers. The following options
-   * are available:
-   *   'default' -- Trace context will be propagated for incoming requests that
-   *   contain the context header. A new trace will be created for requests
-   *   without trace context headers. All traces are still subject to local
-   *   sampling and url filter policies.
-   *   'require' -- Same as default, but traces won't be created for requests
-   *   without trace context headers. This should not be set for end user-facing
-   *   services, as this header is usually set by other traced services rather
-   *   than by users.
-   *   'ignore' -- Trace context headers will always be ignored, so a new trace
-   *   with a unique ID will be created for every request. This means that a
-   *   sampling decision specified on an incoming request will be ignored.
+   * Specifies whether to trace based on the 'traced' bit specified on incoming
+   * trace context headers. The following options are available:
+   *   'default' -- Don't trace incoming requests that have a trace context
+   *   header with its 'traced' bit set to 0.
+   *   'require' -- Don't trace incoming requests that have a trace context
+   *   header with its 'traced' bit set to 0, or incoming requests without a
+   *   trace context header.
+   *   'ignore' -- The 'traced' bit will be ignored. In other words, the context
+   *   header isn't used to determine whether a request will be traced at all.
    *   This might be useful for aggregating traces generated by different cloud
    *   platform projects.
-   * All traces are still subject to local tracing policy.
+   * Note that setting `propagateTraceContextFromHeader` does not change the
+   * behavior of this field, as it controls whether trace context will be
+   * inherited from the incoming request.
    */
   contextHeaderBehavior?: ContextHeaderBehavior;
 
+  /**
+   * For advanced usage only.
+   * If specified, overrides the built-in trace policy object. Note that
+   * ignoreUrls, ignoreMethods, samplingRate, and contextHeaderBehavior will be
+   * ignored if this field is specified.
```

Comment:
Add a throw if any of ignored fields are provided along with `tracePolicy`.

Reply:
I've made it a warning... AFAICT we try not to throw, right?

Reply:
This clearly indicates user error. This is a good example for when one should throw.

Reply:
Ok, makes sense. I've changed it to throw instead.